packages/headless/src/api/analytics/analytics-relay-client.ts

```diff
@@ -28,7 +28,7 @@ export const getRelayInstanceFromState = createSelector(
       url:
         apiBaseUrl ?? getAnalyticsNextApiBaseUrl(organizationId, environment),
       token,
-      trackingId,
+      trackingId: trackingId ?? null,
```

Comment:
Rather than setting null in multiple places, would it be possible to set it once in state as the default value, and have code forward whatever is there?

Reply:
I took this approach to not have to carry over a null everywhere. It seems better to omit the definition of a property as this feels more natural in JS, and handle this corner case at the moment of calling Relay, than to have to ensure everywhere that it's `null`.

Reply:
Resolving based on this [comment](https://github.com/coveo/ui-kit/pull/5316/files#r2089290575)